src/precompile.jl

```diff
@@ -1,7 +1,25 @@
 function _precompile_()
     ccall(:jl_generating_output, Cint, ()) == 1 || return nothing
-    precompile(Tuple{typeof(Parsers.parse), Type{Int64}, String})
-    precompile(Tuple{typeof(Parsers.parse), Type{Float64}, String})
-    precompile(Tuple{typeof(Parsers.parse), Type{Date}, String})
+
+    options = Parsers.Options()
+    pos = 1
+    val = "123"
+    len = length(val)
+    for T in (String, Int32, Int64, Float64, BigFloat, Dates.Date, Dates.DateTime, Bool)
+        for buf in (codeunits(val), Vector(codeunits(val)))
+            Parsers.xparse(T, buf, pos, len, options)
+            if !(T === String)
+                Parsers.xparse(T, buf, pos, len, options, T)
+            end
+            Parsers.xparse(T, buf, pos, len, options, Any)
```

Comment:
why is this method important to precompile? is it because it is used in CSV (as `_parseany`)?

Reply:
Yep for `_parseany`.